lib/src/interactions.dart

```diff
@@ -263,10 +269,22 @@ class Interactions implements IInteractions {
   @override
   Future<void> deleteGlobalCommand(Snowflake commandId) => interactionsEndpoints.deleteGlobalCommand(client.appId, commandId);
 
+  /// Deletes all global commands
+  @override
+  Future<void> deleteGlobalCommands() async => interactionsEndpoints.bulkOverrideGlobalCommands(client.appId, []);
+
   /// Deletes guild command
   @override
   Future<void> deleteGuildCommand(Snowflake commandId, Snowflake guildId) => interactionsEndpoints.deleteGuildCommand(client.appId, commandId, guildId);
 
+  /// Deletes all guild commands for the specified guilds
+  @override
+  Future<void> deleteGuildCommands(List<Snowflake> guildIds) async {
+    await Future.wait(
+      guildIds.map((guildId) => interactionsEndpoints.bulkOverrideGuildCommands(client.appId, guildId, []))
```

Comment:
```suggestion
      guildIds.map((guildId) => interactionsEndpoints.bulkOverrideGuildCommands(client.appId, guildId, []).toList())
```
you gotta do something likt that i guess